### PR TITLE
Add desc of RedisInsight

### PR DESCRIPTION
**Pull request by kaitlynmichaelredislabs**

### CODE REVIEW

**Review by banker — CHANGES_REQUESTED** [review]
Formally requesting changes, as discussed :)

File: content/ri/_index.md [hunk 1]
```diff
@@ -6,7 +6,20 @@ weight: 90
 alwaysopen: false
 categories: ["RI"]
 ---
-Choose the topic you need help with:
+RedisInsight is a free GUI for Redis that is available on all platforms (Windows, Mac, Linux, and Docker) and works with all variants of Redis. RedisInsight allows you to:
+- View performance metrics for your redis instance with the [Overview]({{<relref "/ri/using-redisinsight/overview.md">}}) tool
```
Comment (banker) [inline content/ri/_index.md:10]:
```suggestion
- View performance metrics for your Redis instance with the [Overview tool]({{<relref "/ri/using-redisinsight/overview.md">}})
```

File: content/ri/_index.md [hunk 2]
```diff
@@ -6,7 +6,20 @@ weight: 90
...
+- View data structures visually with the [Browser]({{<relref "/ri/using-redisinsight/browser.md">}}) tool
```
Comment (banker) [inline content/ri/_index.md:11]:
```suggestion
- Visualize data structures with the [Browser tool]({{<relref "/ri/using-redisinsight/browser.md">}})
```

File: content/ri/_index.md [hunk 3]
```diff
@@ -6,7 +6,20 @@ weight: 90
...
+- Manage basic properties of your Redis cluster such as cluster node timeout, IP, or port with the [Cluster Management]({{<relref "/ri/using-redisinsight/cluster-management.md">}}) tool
```
Comment (banker) [inline content/ri/_index.md:12]:
```suggestion
- Manage basic properties of your Redis cluster such as cluster node timeout, IP, or port with the [Cluster Management tool]({{<relref "/ri/using-redisinsight/cluster-management.md">}})
```

File: content/ri/_index.md [hunk 4]
```diff
@@ -6,7 +6,20 @@ weight: 90
...
+- Use commands with a REPL (read-eval-print-loop) with the the [CLI]({{<relref "/ri/using-redisinsight/cli.md" >}}) tool
```
Comment (banker) [inline content/ri/_index.md:13]:
```suggestion
- Run commands with a REPL (read-eval-print-loop) with the the [CLI tool]({{<relref "/ri/using-redisinsight/cli.md" >}})
```

File: content/ri/_index.md [hunk 5]
```diff
@@ -6,7 +6,20 @@ weight: 90
...
+- Analyze memory usage with the [Memory Analysis]({{<relref "/ri/using-redisinsight/memory-analysis.md">}}) tool
+- Identify and troubleshoot bottlenecks with the [Slowlog]({{<relref "/ri/using-redisinsight/slowlog.md">}}) tool
+- Edit the configuration of your Redis instance with the [Configuration]({{<relref "/ri/using-redisinsight/configuration.md">}}) tool
+
+RedisInsight also has support for several Redis modules, including RedisGraph, RedisTimeSeries, and RediSearch.
```
Comment (banker) [inline content/ri/_index.md:18]: Can you link to these in the docs?

File: content/ri/_index.md [hunk 6]
```diff
@@ -6,7 +6,20 @@ weight: 90
...
 ---
-Choose the topic you need help with:
+RedisInsight is a free GUI for Redis that is available on all platforms (Windows, Mac, Linux, and Docker) and works with all variants of Redis. RedisInsight allows you to:
+- View performance metrics for your redis instance with the [Overview]({{<relref "/ri/using-redisinsight/overview.md">}}) tool
+- View data structures visually with the [Browser]({{<relref "/ri/using-redisinsight/browser.md">}}) tool
+- Manage basic properties of your Redis cluster such as cluster node timeout, IP, or port with the [Cluster Management]({{<relref "/ri/using-redisinsight/cluster-management.md">}}) tool
+- Use commands with a REPL (read-eval-print-loop) with the the [CLI]({{<relref "/ri/using-redisinsight/cli.md" >}}) tool
+- Analyze memory usage with the [Memory Analysis]({{<relref "/ri/using-redisinsight/memory-analysis.md">}}) tool
+- Identify and troubleshoot bottlenecks with the [Slowlog]({{<relref "/ri/using-redisinsight/slowlog.md">}}) tool
+- Edit the configuration of your Redis instance with the [Configuration]({{<relref "/ri/using-redisinsight/configuration.md">}}) tool
```
Comment (banker) [inline content/ri/_index.md:16]: Can you include "tool" in the hyperlinks?

File: content/ri/_index.md [hunk 7]
```diff
@@ -6,7 +6,20 @@ weight: 90
...
+
+RedisInsight also has support for several Redis modules, including RedisGraph, RedisTimeSeries, and RediSearch.
+
+
+**Start using RedisInsight:**
```
Comment (banker) [inline content/ri/_index.md:21]:
```suggestion
## Start using RedisInsight
```

**Review by banker — CHANGES_REQUESTED** [review]
Can you make these changes?

File: content/ri/_index.md [hunk 8]
```diff
@@ -6,7 +6,19 @@ weight: 90
...
+- View data structures visually with the [Browser]({{<relref "/ri/using-redisinsight/browser.md">}}) tool
+- Manage basic properties of your Redis cluster such as cluster node timeout, IP, or port with the [Cluster Management tool]({{<relref "/ri/using-redisinsight/cluster-management.md">}})
+- Run commands with a REPL (read-eval-print-loop) with the the [CLI tool]({{<relref "/ri/using-redisinsight/cli.md" >}})
+- Analyze memory usage with the [Memory Analysis]({{<relref "/ri/using-redisinsight/memory-analysis.md">}}) tool
+- Identify and troubleshoot bottlenecks with the [Slowlog]({{<relref "/ri/using-redisinsight/slowlog.md">}}) tool
+- Edit the configuration of your Redis instance with the [Configuration]({{<relref "/ri/using-redisinsight/configuration.md">}}) tool
 
-- [Installing RedisInsight]({{< relref "/ri/installing/_index.md" >}})
-- [Using RedisInsight]({{< relref "/ri/using-redisinsight/add-instance.md" >}})
+RedisInsight also has support for several Redis modules, including RedisGraph, RedisTimeSeries, and RediSearch.
```
Comment (kaitlynmichaelredislabs) [inline content/ri/_index.md:18]:
```suggestion
RedisInsight also has support for several Redis modules, including [RedisGraph]({{<relref "/modules/redisgraph/_index.md">}}), [RedisTimeSeries]({{<relref "/modules/redistimeseries/_index.md">}}), and [RediSearch]({{<relref "/modules/redisearch/_index.md">}})
```

**Review by banker — APPROVED** [review]

**Pull request merged**